src/QMCWaveFunctions/Jastrow/PadeFunctors.h

```diff
@@ -60,14 +61,19 @@ struct PadeFunctor:public OptimizableFunctorBase
   std::string ID_B;
 
   ///default constructor
-  PadeFunctor(): Scale(1.0),ID_A("0"),ID_B("0") { }
+  PadeFunctor(): Scale(1.0),ID_A("0"),ID_B("0"),A(1.0),B0(1.0),Opt_A(false),Opt_B(true) 
+  { 
+    reset();
+  }
 
+  /*
```

Comment:
There are a few commented builder. Do we still need them?

Reply:
They are no longer needed, I have removed them